install-browser.mjs

```diff
@@ -0,0 +1,44 @@
+/**
```

Comment:
Not a fan of adding this file here...shouldn't `@puppeteer/browsers` be responsible for this logic instead?

Reply:
It is responsible but requires an integration with GitHub Actions. Therefore a cross-platform Node script is preferrable to sh/bash.

Reply:
Still, it would be more ideal to e.g. have @puppeteer/browsers provide an action we can `uses:` here, with custom inputs (browser + version + cache dir).

I am OK to land this as an intermediate step, but long-term we should get rid of this file _in this repo_, IMHO.

Reply:
Sounds good to me!